frontend/src/components/table/schedule/schedule.component.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect } from "react";
 import { useSelector } from "react-redux";
 import { ApplicationStateModel } from "../../../state/models/application-state.model";
-import { WorkerType } from "../../../state/models/schedule-data/employee-info.model";
+import { WorkerType } from "../../../state/models/schedule-data/worker-info.model";
```

Comment:
Pomyślałem, że lepiej jest zamienić `employee` na workers, ponieważ oznaczają to samo, a `workers ` jest wykorzystywany w znacznie większej ilości miejsc

Reply:
W pełni się zgadzam, zdecydowanie lepiej brzmi